### PR TITLE
DOC: update fiona / shapely links

### PR DESCRIPTION
**Pull request by Casyfill**

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@ operations are performed by `shapely`_.  Geopandas further depends on
 `fiona`_ for file access and `descartes`_ and `matplotlib`_ for plotting.
 
 .. _pandas: http://pandas.pydata.org
-.. _shapely: http://toblerity.github.io/shapely
-.. _fiona: http://toblerity.github.io/fiona
+.. _shapely: https://shapely.readthedocs.io/
+.. _fiona: http://toblerity.org/shapely
```
Comment (jorisvandenbossche) [inline doc/source/index.rst]: this one should be fiona

Comment (Casyfill) [reply doc/source/index.rst]: Oh my

**Comment by Casyfill** [comment]
done

**Comment by jorisvandenbossche** [comment]
@Casyfill Thanks!

**Pull request merged**